pkg/components/simplejson/simplejson.go

```diff
@@ -445,26 +432,3 @@ func (j *Json) MustInt64(args ...int64) int64 {
 
 	return def
 }
-
-// MustUInt64 guarantees the return of an `uint64` (with optional default)
-//
-// useful when you explicitly want an `uint64` in a single value return context:
-//     myFunc(js.Get("param1").MustUint64(), js.Get("optional_param").MustUint64(5150))
-func (j *Json) MustUint64(args ...uint64) uint64 {
-	var def uint64
-
-	switch len(args) {
-	case 0:
-	case 1:
-		def = args[0]
-	default:
-		log.Panicf("MustUint64() received too many arguments %d", len(args))
-	}
-
-	i, err := j.Uint64()
-	if err == nil {
-		return i
-	}
-
-	return def
-}
```

Comment:
this is more like an embedded library, just because we do not use these functions yet seems unnecessary to remove them

Reply:
@torkelo OK, so to clarify, I should undo all simplejson changes in this PR? And flag all of the simplejson code as in-use (so the linter doesn't flag them again)?

Reply:
yea, think so. Or vendor in https://github.com/bitly/go-simplejson  but not sure if we have made changes and these have diverged, can't remember why I just copied that lib and did not vendor it

Reply:
OK, thanks!